src/main/kotlin/com/vk/admstorm/env/Env.kt

```diff
@@ -128,6 +130,15 @@ object Env {
             LOG.warn("Exception while deserialize data, further work of the plugin is impossible:", e)
             return
         }
+        val config = AdmStormSettingsState.getInstance()
+        if (config.localDeployConfig) {
+            val publishConfig = project.getService(PublishConfig::class.java)
+            val defServer = publishConfig.findDefaultServers().firstOrNull()
+            if (defServer != null) {
+                data.projectRoot = defServer.fileTransferConfig.rootFolder
```

Comment:
what should we do if the `rootFolder` is empty?

Reply:
It seems to me that this is already some kind of implicitness, there will be an error when connecting and the user would uncheck the box in the settings, but okay